.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG IMAGE="python:3.12"
```

Comment:
can we support python3.13

Reply:
Sure

Reply:
no we can't it's not available yet in the container image.
https://github.com/devcontainers/images/tree/main/src/python

Can we merge this now so I can merge the other one?

Reply:
Last thing... Can you please update the PR description with a walkthrough of the changes and why you made them?

Reply:
done
can we merge now?